gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -338,45 +339,50 @@ public void translate (float xAmount, float yAmount) {
 	/** Sets the color used to tint this sprite. Default is {@link Color#WHITE}. */
 	public void setColor (Color tint) {
 		color.set(tint);
-		float color = tint.toFloatBits();
+		packedColor = tint.toFloatBits();
 		float[] vertices = this.vertices;
-		vertices[C1] = color;
-		vertices[C2] = color;
-		vertices[C3] = color;
-		vertices[C4] = color;
+		vertices[C1] = packedColor;
```

Comment:
We already store the packed color in the vertices, though using it (as we used to) is a bit hacky.